helm/templates/_helpers.tpl

```diff
@@ -24,16 +24,18 @@ Get the current recommended cloudwatch agent image for a region
 {{- printf "%s/%s:%s" $imageDomain .Values.agent.image.repository .Values.agent.image.tag -}}
 {{- end -}}
 
+{{/*
+Get the current recommended cloudwatch agent image for a region
+*/}}
+{{- define "cloudwatch-agent-windows.image" -}}
+{{- default "506463145083.dkr.ecr.us-west-2.amazonaws.com/windows-container-internal:latest" -}}
+{{- end -}}
+
 {{/*
 Get the current recommended cloudwatch agent operator image for a region
 */}}
 {{- define "cloudwatch-agent-operator.image" -}}
-{{- $imageDomain := "" -}}
```

Comment:
Instead of changing the existing one, can we define a cloudwatch operator windows image to use

Reply:
Anyway this is supposed to be temporary change just for testing before operator with my changes is released. We will fix this before it gets merged to `main`